src/batch-client/index.ts

```diff
@@ -614,6 +626,18 @@ export class ZimbraBatchClient {
 			body: denormalize(GetRightsRequest)(options)
 		}).then(normalize(AccountRights));
 
+	public getScratchCodes = (username: String) =>
+		this.jsonRequest({
+			name: 'GetScratchCodes',
+			namespace: Namespace.Account,
+			body: {
```

Comment:
Is this payload required ? API doesn't say so...

Also, I doubt `namespace` is required here.

Reply:
yes, I tried this api without this payload, it won't work so i checked the api call that is being made by legacy which includes this payload. So without the account name this api does not work as expected

Reply:
the default namespace is mail, and this call is related to account so i added the namespace

Reply:
https://files.zimbra.com/docs/soap_api/8.8.8/api-reference/index.html , the namesapace mentioned is zimbraAccount